README.md

```diff
@@ -14,10 +14,12 @@ A curated list of awesome [aeternity](https://aeternity.com) resources
 - [twitter/aeternity](https://twitter.com/aeternity) - Official twitter account
 - [twitter/aeternityTECH](https://twitter.com/aeternityTECH) - Tech updates
 
-## Node and middleware
+## Protocol
+- [Protocol specification](https://github.com/aeternity/protocol) - Documentation about the the aeternty protocol
 
+## Node and middleware
 - [Node](https://github.com/aeternity/aeternity) - The official aeternity node written in erlang
-- [Protocol specification](https://github.com/aeternity/protocol) - Documentation about the the aeternty protocol
+- [Node documentation](http://docs.aeternity.io/en/stable/) - Documentation about operating a the official node implementation
```

Comment:
I would link to `http://docs.aeternity.io/` instead so that the default version can be controller by the docs service, also the user language (even we support only en now).